setup/azurecmdfiles.wxi

```diff
@@ -364,9 +364,18 @@
                         </Component>
                     </Directory>
                     <Directory Id="dir8A823C7DB199DCB444D0A6388BC3C769" Name="AzureRM.AzureStackStorage">
+                        <Component Id="cmpC180F2B7917E4BFE6CA02D4CE6A32058" Guid="*">
```

Comment:
why are these files added in the installer?  You should not require any resource manager cmdlets modules as part of this module